src/frontend/citizen-portal/src/app/components/disputant-form/disputant-form.component.ts

```diff
@@ -170,20 +165,24 @@ export class DisputantFormComponent implements OnInit, AfterViewInit {
   onDLProvinceChange(province: ProvinceCodeValue) {
     setTimeout(() => {
       let form = this.form as NoticeOfDisputeFormGroup;
-      form.controls.drivers_licence_province.setValue(province.provAbbreviationCd);
-      form.controls.drivers_licence_country_id.setValue(province.ctryId);
-      form.controls.drivers_licence_province_seq_no.setValue(province.provSeqNo);
 
-      if (province.provId === this.bc.provId) {
+      if (province === null) {
```

Comment:
Should this just be `if (!province) {`    `province` could be null or undefined?

Reply:
I think it should remain as `if (province === null) {` because there is an option added to select blank province now for driver's licence province and its value is set to `null` if selected.